File: 935.c
#include <stdio.h>
 
int main(){
     
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);
     
    if((a+c) % 2 == 0 && (b+d) % 2 == 0 || (a+c) % 2 == 1 && (b+d) % 2 == 1)
        printf("YES");
    else printf("NO");
     
    return 0;
}

File: 935.py
k = input().split()
 
for i in range(4):
    k[i] = int(k[i])
 
if (k[0] + k[2]) % 2 == 0 and (k[1] + k[3]) % 2 == 0 or (k[0] + k[2]) % 2 == 1 and (k[1] + k[3]) % 2 == 1:
    print("YES")
else:
    print("NO")
